src/kaboom.ts

```diff
@@ -369,11 +288,7 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
     }
 
     // create a <canvas> if user didn't provide one
-    const canvas = gopt.canvas ?? (() => {
-        const canvas = document.createElement("canvas");
-        root.appendChild(canvas);
-        return canvas;
-    })();
+    const canvas = gopt.canvas ?? root.appendChild(document.createElement("canvas"))
```

Comment:
Not related to PR, I'll let it go this time

Reply:
Yeah, that's fine, commited it and wasn't sure how to separate to different PR after.